[06Sep2022 00:04:44.316] [Server thread/INFO] [net.minecraft.network.login.ServerLoginNetHandler/]: com.mojang.authlib.GameProfile@55f12837[id=<null>,name=Nave808,properties={},legacy=false] (/192.168.1.179:55646) lost connection: Disconnected
[06Sep2022 00:04:51.259] [User Authenticator #7/INFO] [net.minecraft.network.login.ServerLoginNetHandler/]: UUID of player Nave808 is edaf0a37-e3b3-490a-8626-dbfac2b1f3f7
[06Sep2022 00:05:02.989] [Server thread/INFO] [net.minecraftforge.common.AdvancementLoadFix/]: Using new advancement loading for net.minecraft.advancements.PlayerAdvancements@2ecc62a4
[06Sep2022 00:05:03.070] [Server thread/INFO] [net.minecraft.server.management.PlayerList/]: Nave808[/192.168.1.179:54154] logged in with entity id 35726 at (-1674.146104303533, 32.5, 1386.3249077474607)
[06Sep2022 00:05:03.077] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 joined the game
[06Sep2022 00:05:03.084] [Server thread/INFO] [Waila/]: Syncing config to Nave808 (edaf0a37-e3b3-490a-8626-dbfac2b1f3f7)
[06Sep2022 00:06:07.363] [Server thread/INFO] [net.minecraft.network.play.ServerPlayNetHandler/]: Nave808 lost connection: Disconnected
[06Sep2022 00:06:07.363] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 left the game
[06Sep2022 00:10:24.247] [User Authenticator #8/INFO] [net.minecraft.network.login.ServerLoginNetHandler/]: UUID of player Nave808 is edaf0a37-e3b3-490a-8626-dbfac2b1f3f7
[06Sep2022 00:10:36.137] [Server thread/INFO] [net.minecraftforge.common.AdvancementLoadFix/]: Using new advancement loading for net.minecraft.advancements.PlayerAdvancements@74b23ae5
[06Sep2022 00:10:36.218] [Server thread/INFO] [net.minecraft.server.management.PlayerList/]: Nave808[/192.168.1.179:34548] logged in with entity id 35841 at (-1674.146104303533, 32.5, 1386.3249077474607)
[06Sep2022 00:10:36.224] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 joined the game
[06Sep2022 00:10:36.230] [Server thread/INFO] [Waila/]: Syncing config to Nave808 (edaf0a37-e3b3-490a-8626-dbfac2b1f3f7)
[06Sep2022 00:14:10.241] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 opened a Vault!
[06Sep2022 00:14:16.598] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 entered an Afterlife, Hunger, Plentiful, Stronk Scavenger Vault!
[06Sep2022 00:15:30.563] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Successfully filled 1 blocks]
[06Sep2022 00:15:36.644] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Successfully filled 9784 blocks]
[06Sep2022 00:15:36.644] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Successfully filled 1 blocks]
[06Sep2022 00:15:48.585] [Server thread/WARN] [net.minecraft.server.management.PlayerInteractionManager/]: Mismatch in destroy block pos: BlockPos{x=1797917, y=8, z=4094} BlockPos{x=1797919, y=8, z=4093}
[06Sep2022 00:16:09.349] [Server thread/WARN] [net.minecraft.server.management.PlayerInteractionManager/]: Mismatch in destroy block pos: BlockPos{x=1797896, y=7, z=4102} BlockPos{x=1797895, y=7, z=4100}
[06Sep2022 00:36:32.280] [Worker-Main-2551/WARN] [net.minecraft.entity.item.ItemFrameEntity/]: Unable to load item from: {id:"the_vault:janitors_broom",Count:1b,tag:{RepairCost:0,Damage:0,display:{Name:'{"text":"Douwsky\'s Janitor\'s Broom"}'}}}
[06Sep2022 00:36:32.282] [Worker-Main-2551/WARN] [net.minecraft.entity.EntityType/]: Skipping Entity with id 
[06Sep2022 00:39:02.162] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 bailed.
[06Sep2022 00:44:50.596] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 opened a Vault!
[06Sep2022 00:44:56.859] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 entered an Afterlife, Optimistic, Easy, Weakened, Lucky Scavenger Vault!
[06Sep2022 00:45:00.548] [Worker-Main-2614/WARN] [net.minecraft.entity.item.ItemFrameEntity/]: Unable to load item from: {id:"the_vault:janitors_broom",Count:1b,tag:{RepairCost:0,Damage:0,display:{Name:'{"text":"Douwsky\'s Janitor\'s Broom"}'}}}
[06Sep2022 00:45:00.548] [Worker-Main-2614/WARN] [net.minecraft.entity.EntityType/]: Skipping Entity with id 
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a DUMMY block entity @ BlockPos{x=1806409, y=40, z=3694} but found not block entity block Block{minecraft:air} at location
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a block entity for block Block{minecraft:air} but failed at location BlockPos{x=1806409, y=40, z=3694}
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a DUMMY block entity @ BlockPos{x=1806409, y=40, z=3693} but found not block entity block Block{minecraft:air} at location
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a block entity for block Block{minecraft:air} but failed at location BlockPos{x=1806409, y=40, z=3693}
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a DUMMY block entity @ BlockPos{x=1806409, y=41, z=3693} but found not block entity block Block{minecraft:air} at location
[06Sep2022 00:45:35.114] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a block entity for block Block{minecraft:air} but failed at location BlockPos{x=1806409, y=41, z=3693}
[06Sep2022 00:46:56.761] [IO-Worker-2630/WARN] [net.minecraft.world.chunk.storage.RegionFile/]: Saving oversized chunk [112896, 256] (1388984 bytes} to external file ./Vault-Hunters/dimensions/the_vault/vault/region/c.112896.256.mcc
[06Sep2022 00:48:46.993] [Server thread/WARN] [net.minecraft.server.management.PlayerInteractionManager/]: Mismatch in destroy block pos: BlockPos{x=1806329, y=14, z=3561} BlockPos{x=1806329, y=15, z=3561}
[06Sep2022 00:48:47.427] [Server thread/WARN] [net.minecraft.server.management.PlayerInteractionManager/]: Mismatch in destroy block pos: BlockPos{x=1806327, y=17, z=3562} BlockPos{x=1806329, y=15, z=3561}
[06Sep2022 00:49:47.575] [Worker-Main-2668/WARN] [net.minecraft.entity.item.ItemFrameEntity/]: Unable to load item from: {id:"the_vault:janitors_broom",Count:1b,tag:{RepairCost:0,Damage:0,display:{Name:'{"text":"Douwsky\'s Janitor\'s Broom"}'}}}
[06Sep2022 00:49:47.575] [Worker-Main-2668/WARN] [net.minecraft.entity.EntityType/]: Skipping Entity with id 
[06Sep2022 00:58:16.363] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Played sound minecraft:entity.generic.explode to Nave808]
[06Sep2022 00:58:32.219] [Server thread/WARN] [net.minecraft.world.TrackedEntity/]: Fetching packet for removed entity ItemEntity['Benitoite Ore'/97315, l='ServerLevel[Vault-Hunters]', x=1805755.72, y=38.50, z=4234.38]
[06Sep2022 01:00:01.832] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 01:00:01.833] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world---
[06Sep2022 01:00:02.833] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now disabled]
[06Sep2022 01:00:03.977] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Saved the game]
[06Sep2022 01:00:04.978] [RCON Client /192.168.1.179 #5/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 01:01:55.072] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a DUMMY block entity @ BlockPos{x=1806118, y=27, z=4510} but found not block entity block Block{minecraft:air} at location
[06Sep2022 01:01:55.073] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a block entity for block Block{minecraft:air} but failed at location BlockPos{x=1806118, y=27, z=4510}
[06Sep2022 01:01:55.073] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a DUMMY block entity @ BlockPos{x=1806119, y=26, z=4510} but found not block entity block Block{minecraft:air} at location
[06Sep2022 01:01:55.073] [Server thread/WARN] [net.minecraft.world.chunk.Chunk/]: Tried to load a block entity for block Block{minecraft:air} but failed at location BlockPos{x=1806119, y=26, z=4510}
[06Sep2022 01:02:35.537] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 01:02:35.537] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now enabled]
[06Sep2022 01:02:36.537] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world complete---
[06Sep2022 01:02:36.538] [RCON Client /192.168.1.179 #6/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 01:06:14.436] [Server thread/WARN] [net.minecraft.network.play.ServerPlayNetHandler/]: Nave808 moved wrongly!
[06Sep2022 01:08:54.763] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Summoned new Primed TNT]
[06Sep2022 01:10:46.313] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [@: Summoned new Primed TNT]
[06Sep2022 01:12:41.116] [Server thread/WARN] [net.minecraft.world.TrackedEntity/]: Fetching packet for removed entity ItemEntity['Redstone Ore'/123609, l='ServerLevel[Vault-Hunters]', x=1806326.50, y=33.32, z=4055.60]
[06Sep2022 01:13:17.313] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 bailed.
[06Sep2022 01:13:36.955] [IO-Worker-2891/WARN] [net.minecraft.world.chunk.storage.RegionFile/]: Saving oversized chunk [112896, 256] (1388984 bytes} to external file ./Vault-Hunters/dimensions/the_vault/vault/region/c.112896.256.mcc
[06Sep2022 01:17:18.936] [Server thread/FATAL] [net.minecraft.util.concurrent.ThreadTaskExecutor/]: Error executing task on Server
java.lang.RuntimeException: Slot 22 not in valid range - [0,22)
	at net.minecraftforge.items.ItemStackHandler.validateSlotIndex(ItemStackHandler.java:221) ~[forge:?]
	at net.minecraftforge.items.ItemStackHandler.insertItem(ItemStackHandler.java:87) ~[forge:?]
	at invtweaks.util.Utils.condensed(Utils.java:187) ~[invtweaks:?]
	at invtweaks.util.Sorting.executeSort(Sorting.java:240) ~[invtweaks:?]
	at invtweaks.packets.PacketSortInv.lambda$handle$0(PacketSortInv.java:25) ~[invtweaks:?]
	at net.minecraftforge.fml.network.NetworkEvent$Context.enqueueWork(NetworkEvent.java:215) ~[forge:?]
	at invtweaks.packets.PacketSortInv.handle(PacketSortInv.java:22) ~[invtweaks:?]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.lambda$tryDecode$3(IndexedMessageCodec.java:128) ~[forge:?]
	at java.util.Optional.ifPresent(Optional.java:159) ~[?:1.8.0_342]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.tryDecode(IndexedMessageCodec.java:128) ~[forge:?]
	at net.minecraftforge.fml.network.simple.IndexedMessageCodec.consume(IndexedMessageCodec.java:162) ~[forge:?]
	at net.minecraftforge.fml.network.simple.SimpleChannel.networkEventListener(SimpleChannel.java:80) ~[forge:?]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.fml.network.NetworkInstance.dispatch(NetworkInstance.java:86) ~[forge:?]
	at net.minecraftforge.fml.network.NetworkHooks.lambda$onCustomPayload$1(NetworkHooks.java:91) ~[forge:?]
	at java.util.Optional.map(Optional.java:215) ~[?:1.8.0_342]
	at net.minecraftforge.fml.network.NetworkHooks.onCustomPayload(NetworkHooks.java:91) ~[forge:?]
	at net.minecraft.network.play.ServerPlayNetHandler.func_147349_a(ServerPlayNetHandler.java:1413) ~[?:?]
	at net.minecraft.network.play.client.CCustomPayloadPacket.func_148833_a(CCustomPayloadPacket.java:42) ~[?:?]
	at net.minecraft.network.play.client.CCustomPayloadPacket.func_148833_a(CCustomPayloadPacket.java:12) ~[?:?]
	at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?]
	at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?]
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:734) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:159) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:717) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:711) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213161_c(SourceFile:127) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:697) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:646) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_342]
[06Sep2022 01:58:20.817] [Server thread/INFO] [net.minecraft.network.play.ServerPlayNetHandler/]: Nave808 lost connection: Disconnected
[06Sep2022 01:58:20.817] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Nave808 left the game
[06Sep2022 05:00:01.605] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 05:00:01.605] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world---
[06Sep2022 05:00:02.606] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now disabled]
[06Sep2022 05:00:03.649] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Saved the game]
[06Sep2022 05:00:04.650] [RCON Client /192.168.1.179 #7/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 05:01:05.741] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 05:01:05.742] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now enabled]
[06Sep2022 05:01:06.742] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world complete---
[06Sep2022 05:01:06.743] [RCON Client /192.168.1.179 #8/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 09:00:00.381] [Server thread/INFO] [iskallia.vault.Vault/]: Reset SoulShard Trades!
[06Sep2022 09:00:01.805] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 09:00:01.806] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world---
[06Sep2022 09:00:02.806] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now disabled]
[06Sep2022 09:00:03.851] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Saved the game]
[06Sep2022 09:00:04.851] [RCON Client /192.168.1.179 #9/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 09:00:15.710] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 09:00:15.710] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now enabled]
[06Sep2022 09:00:16.710] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world complete---
[06Sep2022 09:00:16.711] [RCON Client /192.168.1.179 #10/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
[06Sep2022 12:00:01.763] [RCON Listener #2/INFO] [net.minecraft.network.rcon.RConThread/]: Thread RCON Client /192.168.1.179 started
[06Sep2022 12:00:01.764] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon] ---saving world---
[06Sep2022 12:00:02.764] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Automatic saving is now disabled]
[06Sep2022 12:00:03.808] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: [Rcon: Saved the game]
[06Sep2022 12:00:04.809] [RCON Client /192.168.1.179 #11/INFO] [net.minecraft.network.rcon.ClientThread/]: Thread RCON Client /192.168.1.179 shutting down
